connect/api/src/test/java/org/apache/kafka/connect/data/ValuesTest.java

```diff
@@ -73,6 +76,15 @@
         INT_LIST.add(-987654321);
     }
 
+    @Test(timeout = 5000)
+    public void shouldNotEncounterInfiniteLoop() {
+        byte[] bytes = new byte[] { -17, -65,  -65 };
```

Comment:
we need a comment here to explain things.

Reply:
👍  good call, will add